docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -21,12 +21,8 @@ GitHub Actions with Teleport can be applied as both a viable and secure solution
 CI/CD platform is needed.
 
 Teleport supports secure joining on both GitHub-hosted and self-hosted GitHub
-actions runners.
-
-<Admonition type="note">
-Currently GitHub Actions with Machine ID is unsupported on GitHub Enterprise
-Server.
-</Admonition>
+Actions runners. GitHub Enterprise Server requires
+[additional configuration](#github-enterprise-server).
```

Comment:
I think we can find a smoother alternative to linking the user to the last section of the page, only to make them scroll up again to start the guide. What would you think of a `Tabs` component with `TabItem`s for:
- "GitHub Enterprise Server" (the instructions in the current final section) 
- "Managed GitHub" (or whatever the best term is for regular GitHub), which includes the `token` config in the current Step 1

Reply:
d620de3401